kratos/processes/find_intersected_geometrical_objects_process.h

```diff
@@ -401,12 +401,6 @@ class KRATOS_API(KRATOS_CORE) FindIntersectedGeometricalObjectsProcess
     ///@name Operations
     ///@{
 
-    /**
-     * @brief This function is designed for being called at the beginning of the computations right after reading the model and the groups
-     * @todo This should be moved to ExecuteInitialize (base class of Process)
-     */
-    virtual void Initialize();
-
```

Comment:
I'd leave this API breaker change for a standalone PR. Note that you're silently breaking the compatibility with everything not covered by the CI.

Reply:
Indeed, I'd first deprecate the method (both in compilation and usage) for a while to then remove it.

Reply:
Ok, will do. I will revert these changes here so the fix can go fast and keep the Initialize changes in another PR.